roles/nvidia-gpu-operator/tasks/nvaie.yml

```diff
@@ -5,7 +5,8 @@
     gpu_operator_enable_migmanager: false
 
 - name: Create namespace for GPU Operator resources
-  shell: kubectl create namespace {{ gpu_operator_namespace }} -o yaml --dry-run=client | kubectl apply -f -
+  shell: set -o pipefail && kubectl create namespace {{ gpu_operator_namespace }} -o yaml --dry-run=client | kubectl apply -f -
+  changed_when: false
```

Comment:
This should be be changed_when: true, shouldn't it?

Reply:
See my high-level comment.